actions/websocket_actions.jsx

```diff
@@ -521,24 +523,24 @@ export function handleEvent(msg) {
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_CREATED:
-        dispatch(handleSidebarCategoryCreated(msg));
+        wrapEventWithJitter(handleSidebarCategoryCreated, msg, dispatch);
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_UPDATED:
-        dispatch(handleSidebarCategoryUpdated(msg));
+        wrapEventWithJitter(handleSidebarCategoryUpdated, msg, dispatch);
         break;
 
     case SocketEvents.SIDEBAR_CATEGORY_DELETED:
-        dispatch(handleSidebarCategoryDeleted(msg));
+        wrapEventWithJitter(handleSidebarCategoryDeleted, msg, dispatch);
         break;
     case SocketEvents.SIDEBAR_CATEGORY_ORDER_UPDATED:
         dispatch(handleSidebarCategoryOrderUpdated(msg));
         break;
     case SocketEvents.USER_ACTIVATION_STATUS_CHANGED:
-        dispatch(handleUserActivationStatusChange());
+        wrapEventWithJitter(handleUserActivationStatusChange, {}, dispatch);
```

Comment:
I know `handleUserActivationStatusChange` doesn't look at the websocket message, but I think we should pass the actual message into these ones instead of an empty object for consistency

Reply:
added the msg for consistency